ckanext/expire_api_token/plugin.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
```

Comment:
This sounds like a really useful feature, any reason to not include it in the core implementation and use a core extension instead?

Reply:
I'm obsessed with extendibility in this PR because I have no ideas what others may expect/want to have. And, if you want to use a calendar widget for selecting the expiration date of token instead of specifying its lifetime, you can just create new extension(of even inherit from `expire_api_token`) with desired fields/validation/widget and use this extension instead of `expire_api_token`. 
If it becomes the part of default implementation, you will have to use existing format (`units` * `duration`) and that leads us to a number of hacks, like using JS and hidden fields for converting date from calendar into the required format

Reply:
I think it's ok as a core extension because it provides a non-toy example of extending API Tokens.